DeployJRE/.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't see a reason to keep this project; all we need is the ant script, right?